message.go

```diff
@@ -1,15 +1,19 @@
 package engo
 
+// MessageHandler is a function that takes a `Message` parameter
```

Comment:
I can already tell that this function takes a message parameter.

Why do I use this function?